### ISSUE TITLE
Publish to Open VSX Registry

### ISSUE THREAD

**Issue by mountainash** [frencojobs/vscode-vercel#6]
I saw the post on [Hasnode](https://frenco.hashnode.dev/vscode-vercel-an-extension-for-your-favorite-code-editor) - looks like this could be helpful. Could this extension please be published to https://open-vsx.org/ swell as Microsoft's Visual Studio Marketplace? (for [easy] use in [VScodium](https://github.com/VSCodium/vscodium)

**Comment by frencojobs** [comment]
I'm not familiar with VSCodium but isn't it possible to install the vsix files in it?

**Comment by mountainash** [comment]
Yes - the same way you can in VSCode. But having it in in the Open VSX market place makes it installable from the Extensions sidebar (much easier).

**Comment by frencojobs** [comment]
Sure, I'll make a release for it in the next update very soon.
